nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -425,249 +215,32 @@ contract StakingEscrow is Issuer, IERC900History {
-    }
-
-    /**
-    * @notice Get the last committed staker's period
-    * @param _staker Staker
-    */
-    function getLastCommittedPeriod(address _staker) public view returns (uint16) {
-        StakerInfo storage info = stakerInfo[_staker];
-        return info.nextCommittedPeriod != 0 ? info.nextCommittedPeriod : info.lastCommittedPeriod;
-    }
-
-    /**
-    * @notice Get the value of locked tokens for active stakers in (getCurrentPeriod() + _offsetPeriods) period
-    * as well as stakers and their locked tokens
-    * @param _offsetPeriods Amount of periods for locked tokens calculation
-    * @param _startIndex Start index for looking in stakers array
-    * @param _maxStakers Max stakers for looking, if set 0 then all will be used
-    * @return allLockedTokens Sum of locked tokens for active stakers
-    * @return activeStakers Array of stakers and their locked tokens. Stakers addresses stored as uint256
-    * @dev Note that activeStakers[0] in an array of uint256, but you want addresses. Careful when used directly!
-    */
-    function getActiveStakers(uint16 _offsetPeriods, uint256 _startIndex, uint256 _maxStakers)
-        external view returns (uint256 allLockedTokens, uint256[2][] memory activeStakers)
-    {
-        require(_offsetPeriods > 0);
-
-        uint256 endIndex = stakers.length;
-        require(_startIndex < endIndex);
-        if (_maxStakers != 0 && _startIndex + _maxStakers < endIndex) {
-            endIndex = _startIndex + _maxStakers;
-        }
-        activeStakers = new uint256[2][](endIndex - _startIndex);
-        allLockedTokens = 0;
-
-        uint256 resultIndex = 0;
-        uint16 currentPeriod = getCurrentPeriod();
-        uint16 nextPeriod = currentPeriod.add16(_offsetPeriods);
-
-        for (uint256 i = _startIndex; i < endIndex; i++) {
-            address staker = stakers[i];
-            StakerInfo storage info = stakerInfo[staker];
-            if (info.currentCommittedPeriod != currentPeriod &&
-                info.nextCommittedPeriod != currentPeriod) {
-                continue;
-            }
-            uint256 lockedTokens = getLockedTokens(info, currentPeriod, nextPeriod);
-            if (lockedTokens != 0) {
-                activeStakers[resultIndex][0] = uint256(staker);
-                activeStakers[resultIndex++][1] = lockedTokens;
-                allLockedTokens += lockedTokens;
-            }
-        }
-        assembly {
-            mstore(activeStakers, resultIndex)
-        }
-    }
-
-    /**
-    * @notice Get worker using staker's address
-    */
-    function getWorkerFromStaker(address _staker) external view returns (address) {
-        return stakerInfo[_staker].worker;
     }
 
     /**
     * @notice Get work that completed by the staker
     */
     function getCompletedWork(address _staker) external view returns (uint256) {
-        return stakerInfo[_staker].completedWork;
+        return token.totalSupply();
```

Comment:
Just to double-check: this implies that we consider all pending work in WorkLock to be done, so people that still have ETH locked can claim it back. Correct? In such case, I think this should be included in the newsfragment.

Reply:
✔️